sonic_api.yaml

```diff
@@ -1255,6 +1255,210 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 #----------------------------------------------
+# Tunnel encap object for vxlan and gre tunnels
+#----------------------------------------------
+  '/config/tunnel/encap/{tunnel_type}/{tunnel_name}':
+    put:
+      summary: Create a tunnel
+      description: If a tunnel with this name doesn't exist yet then create a new tunnel with these parameters. If the tunnel already exists, it's attributes will be updated.
```

Comment:
Please mention, which attributes can be modified.

Reply:
I don't see anything which cannot be updated in TunnelEntry object. Exceptions can be added in implementation for lets say if some field updation is dependent on some other resources.